lua/mapvote/server/modules/rtv.lua

```diff
@@ -82,6 +82,12 @@ function RTV.Start()
     MapVote.Start()
 end
 
+function RTV.ResetVotes()
+    for _, ply in pairs( player.GetAll() ) do
```

Comment:
🤓 erm technically
```suggestion
    for _, ply in ipairs( player.GetAll() ) do
```

Reply:
I can also change this to GetHumans instead